editors/sc-ide/core/main.hpp

```diff
@@ -33,6 +34,8 @@
 #include "settings/manager.hpp"
 #include "../widgets/style/style.hpp"
 
+static void findOpenPort(QWebSocketServer& server);
```

Comment:
don't put this in the header. headers are for definitions that should be shared; also, a `static` function should [almost?] never appear in a header, because then its definition will be placed in every TU that includes this header. just move the definition above `main`, or put this declaration above main